src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -211,8 +211,8 @@ func (c *Client) SetGoogleClientInfo(keyval ...string) {
     c.xGoogHeader = []string{gax.XGoogHeader(kv...)}
 }
 
-// LibraryShelfPath returns the path for the shelf resource.
-func LibraryShelfPath(shelfId string) string {
+// ShelfPath returns the path for the shelf resource.
```

Comment:
I've reviewed the entire diff of this PR against origin/master, and all the changes should be relevant to Discovery gen now.